pelicun/model/damage_model.py

```diff
@@ -1296,6 +1304,74 @@ def map_ds(values: np.ndarray, offset: int) -> np.ndarray:
 
             return ds_id
 
+        def parse_scaling_specification(scaling_specification: dict) -> dict:  # noqa: C901
+            """
+            Parse and validate the scaling specification.
+
+            Parameters
+            ----------
+            scaling_specification : dict
+                The scaling specification to be parsed and validated.
+
+            Returns
+            -------
+            dict
+                The parsed and validated scaling specification.
+
+            Raises
+            ------
+            ValueError
+                If the scaling specification is invalid.
+            TypeError
+                If the type of an entry is invalid.
+            """
+            # if there are contents, ensure they are valid.
+            # See docstring for an example of what is expected.
```

Comment:
This reference to the docstring is somewhat ambiguous now that the code is moved to another function. I suggest editing the comment to reference the docstring of the _create_dmg_RVs function specifically.

Please also update that docstring to describe the new format and check the docstrings of other, higher-level functions that use the scaling_specification as an argument.